tests/test_main.py

```diff
@@ -596,7 +601,7 @@ def check_code(self, code, nodes, decorators, check_names):
                     ex = Source.executing(frame)
                     node = ex.node
                 except Exception:
-                    if inst.opname.startswith(('COMPARE_OP', 'IS_OP', 'CALL_', 'LOAD_NAME', 'STORE_SUBSCR')):
+                    if inst.opname.startswith(('COMPARE_OP', 'IS_OP', 'CALL_', 'LOAD_NAME', 'STORE_ATTR', 'STORE_SUBSCR')):
```

Comment:
I don't think this is needed. Checking the attribute name should generally allow disambiguation. But using `in` isn't specific enough, and `==` fails when private attributes are used.

Reply:
But it failed for `xml.dom.minidom` with statement: `self._attrs, self._attrsNS, self._ownerElement = state`

Reply:
Yes, I'm saying a smarter check on attribute names would be able to handle that.

Reply:
Not sure I followed you. Mind working on it?